style.css

```diff
@@ -113,3 +146,570 @@ a {
   color: white;
   margin-top: 1.2rem;
 }
+
+.recent-work-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px 0;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  top: 682px;
+}
```

Comment:
- [ ] You did a great job using flex but as the previous reviewer ask about you you have to use display:grid as it was part of the project requirements. Keep coding 💪